packages/twenty-front/src/pages/settings/data-model/SettingsObjectNewField/SettingsObjectNewFieldConfigure.tsx

```diff
@@ -40,7 +44,9 @@ type SettingsDataModelNewFieldFormValues = z.infer<
 const DEFAULT_ICON_FOR_NEW_FIELD = 'IconUsers';
 
 export const SettingsObjectNewFieldConfigure = () => {
-  const navigate = useNavigate();
+  const navigateApp = useNavigateApp();
+  const navigate = useNavigateSettings();
```

Comment:
navigateSettings (using named exports could avoid this issue)
const { navigateSettings ) = useNavigateSettings (by default the exported function is named). In case of conflicts:
const { navigateSettings: myCustomName } = ...